<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>479</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>761</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnUp">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../resources/arr_up.png</normaloff>../resources/arr_up.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDown">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../resources/arr_down.png</normaloff>../resources/arr_down.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../resources/remove.png</normaloff>../resources/remove.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QTableWidget" name="twgDest">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="cbxStartup">
      <property name="text">
       <string>Start on OS startup</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Selected brac files and directories</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Choose brac directories or files to be added to the synchronizing queue.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTreeView" name="tvwSrc">
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/arr_right.png</normaloff>../resources/arr_right.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnApply">
        <property name="text">
         <string>&amp;Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbxStartup</tabstop>
  <tabstop>tvwSrc</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>twgDest</tabstop>
  <tabstop>btnUp</tabstop>
  <tabstop>btnDown</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnApply</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbxStartup</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Settings</receiver>
   <slot>onCheckBoxStartup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>released()</signal>
   <receiver>Settings</receiver>
   <slot>onButtonAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUp</sender>
   <signal>released()</signal>
   <receiver>Settings</receiver>
   <slot>onButtonUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDown</sender>
   <signal>released()</signal>
   <receiver>Settings</receiver>
   <slot>onButtonDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>764</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>released()</signal>
   <receiver>Settings</receiver>
   <slot>onButtonRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnApply</sender>
   <signal>released()</signal>
   <receiver>Settings</receiver>
   <slot>onButtonApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>released()</signal>
   <receiver>Settings</receiver>
   <slot>onButtonClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>756</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCheckBoxStartup()</slot>
  <slot>onButtonAdd()</slot>
  <slot>onButtonUp()</slot>
  <slot>onButtonDown()</slot>
  <slot>onButtonRemove()</slot>
  <slot>onButtonApply()</slot>
  <slot>onButtonClose()</slot>
 </slots>
</ui>
